api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,27 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
```

Comment:
You are right about `InstitutionPersistence`, it will time out and shut itself down if it doesn't see activity. The persistent actors are the ones where I've added the behavior to shut themselves down. The other actors (event processor and single lar validation) are started at the beginning and remain in memory as long as the application is running. I'm going to push an update to better reflect how `InstitutionPersistence` should be instantiated in the API